src/large.c

```diff
@@ -114,15 +114,15 @@ large_ralloc_no_move_shrink(tsdn_t *tsdn, extent_t *extent, size_t usize) {
 	/* Split excess pages. */
 	if (diff != 0) {
 		extent_t *trail = extent_split_wrapper(tsdn, arena,
-		    &extent_hooks, extent, usize + large_pad, usize, diff,
-		    diff);
+		    &extent_hooks, extent, usize + large_pad, size2index(usize),
+		    diff, NSIZES);
 		if (trail == NULL) {
 			return true;
 		}
 
 		if (config_fill && unlikely(opt_junk_free)) {
 			large_dalloc_maybe_junk(extent_addr_get(trail),
-			    extent_usize_get(trail));
+			    extent_size_get(trail));
```

Comment:
```extent_usize_get()``` can now only be called for extents that have had their szind set.  In this case we've just split off some trailing pages, and that extent has not had its szind set (nor do we want to set it for extents that won't be directly accessed by the application).  The fix is to rename the ```usize``` argument to ```size``` for ```large_dalloc_[maybe_]junk()```.